### DISCUSSION TITLE
Contact?

### DISCUSSION THREAD

**Discussion by nnitch** [synchronizing/mitm#14] [Q&A] [answered]
G'day, I'm looking at using this for a project I have in mind, I just want to ask some questions about this repo.

If you want to contact me, chuck me a message on Twitter: @mitchnaake
I would send you a pm, but I am unable for some odd reason.

I think that you might find my project quite interesting :)

Have a good one, mate!

**Comment by synchronizing** [comment, marked as answer]
I've enabled "Allow message requests from everyone" on my Twitter [@_synchronizing](https://twitter.com/_synchronizing).
